example/ios/Podfile

```diff
@@ -0,0 +1,44 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Also we don't need ios and android folders for the example project

what do you think?

Reply:
Yeah, I agree, in the current version we just support web, so I will delete other unused platforms.